src/operations/update.ts

```diff
@@ -274,7 +274,7 @@ export function makeUpdateStatement(
     op.upsert = options.upsert;
   }
 
-  if (typeof options.multi === 'boolean') {
+  if (options.multi) {
```

Comment:
Note: this should only be sent when true, since it's false by default - was causing some of the versioned api spec tests which weren't expecting `multi: false` to be there to fail.